open-api/rest-catalog-open-api.yaml

```diff
@@ -1537,9 +1622,39 @@ components:
             EmptyResponse:
               $ref: '#/components/examples/ListNamespacesEmptyExample'
 
+    AuthenticationTimeoutResponse:
+      description:
+        Credentials have timed out. If possible, the client should refresh credentials and retry.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorModel'
+          example: {
+            "error": {
+              "message": "Credentials have timed out",
+              "type": "AuthenticationTimeoutException",
+              "code": 419
+            }
+          }
+
+    ServiceUnavailableResponse:
+      description:
+        The service is not ready to handle the request. The client should wait and retry.
```

Comment:
According to the mozilla documentation, if possible, a `Retry-After` header should be used if possible to indicate when the client should retry after: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503

Presumably the HTTP client handles all of this, and it doesn't need to be in the spec, but wanted to call attention to it. Possibly something to add to the description as a note, even if just in a follow up.

Reply:
It doesn't sound like this is required. Maybe we should mention it as optional?

Reply:
Yeah it's not required. Mentioning it as optional seems beneficial to server implementors, particularly in the rate limiting case.

It can be added as a folow-up though.

Reply:
Added "The service may additionally send a Retry-After header to indicate when to retry".